composer.json

```diff
@@ -37,14 +37,15 @@
   "require": {
     "php": "^7.2",
     "laminas/laminas-servicemanager": "^3.3",
-    "laminas/laminas-stdlib": "^3.1"
+    "laminas/laminas-stdlib": "^3.1",
+    "doctrine/persistence": "^2.0"
   },
   "require-dev": {
     "malukenho/docheader": "^0.1.7",
     "phpunit/phpunit": "^8.5.2",
     "phpspec/prophecy": "^1.10",
     "friendsofphp/php-cs-fixer": "^2.9.3",
-    "doctrine/common": "^2.4",
+    "doctrine/doctrine-orm-module": "^3.1",
```

Comment:
why we need `doctrine/doctrine-orm-module` dependency?

Reply:
To execute the test cases. 
I am not very knowledgeable about Doctrine. If this is the wrong dependency, what should it be?

Reply:
`"doctrine/persistence": "^2.0"`  you added in `require` section is enough I think. Also I am not sure if we should add it to required as this is optional and required only if you keep your roles using doctrine. I even thought removing it from this module it is used only in this provider https://github.com/LM-Commons/LmcRbac/blob/master/src/Role/ObjectRepositoryRoleProvider.php

Reply:
We need the dependency in `require-dev` to support the test cases for the `ObjectRepositoryRoleProvider`.
There was a conflict when specifying `persistence v2` and `"doctrine/common": "^2.4"` so I changed it to require what was needed to support the test cases.
I agree that this should be moved outside of `LmcRbac` into a seperate library.  The same issue exists with `LmcRbacMvc`.

Reply:
Yes, I understand why you added. I'm quite familiar with the doctrine. `doctrine/persistence` that was part of doctrine/common before. just saying that `doctrine/persistence` should go to require-dev and `doctrine/doctrine-orm-module` is not required at all I don't know why you added it.